src/mmg3d/split_3d.c

```diff
@@ -1251,21 +1246,22 @@ int MMG5_split2sf_globNum(MMG5_pMesh mesh,MMG5_pSol met,MMG5_int k,MMG5_int vx[6
   MMG5_int            newtet[3];
   int8_t              firstxt,isxt[3];
   uint8_t             tau[4],imin;
-  const uint8_t       *taued;
+  const uint8_t       *taued=NULL;
   const int           ne=3;
 
   pt[0] = &mesh->tetra[k];
   flg   = pt[0]->flag;
   pt[0]->flag = 0;
   newtet[0]=k;
 
+  /* Determine tau, taued and imin the condition for vertices permutation */
```

Comment:
Maybe add a comment to explain that `vGlobNum` is setted to `pt->v` when we come from split2sf and this value may point to a wrong memory address if the tetra array is reallocated before the use of `vGlobNum` (`MMG3D_crea_newTetra` may reallocate the tetra array and `vGlobNum` is used inside `split2sf_cfg`) because the `pt` pointer may become invalid. Thus, it is mandatory to call split*_cfg before newTetra function.

Reply:
Done.